pkgs/development/tools/cdecrypt/default.nix

```
@@ -0,0 +1,24 @@
+{ lib, stdenv, fetchFromGitHub }:
+stdenv.mkDerivation rec {
+  pname = "cdecrypt";
+  version = "4.8";
+
+  src = fetchFromGitHub {
+    owner = "VitaSmith";
+    repo = "cdecrypt";
+    rev = "v${version}";
+    sha256 = "PyT60RDyp1/Co/7WHC0+KrsnrDeTJ605x1pt4OmlGYg=";
```

SRI hashes are preferred nowadays.

```suggestion
    hash = "sha256-.........";
```

---

change applied in rebased commit.